### DISCUSSION TITLE
Duplicate PHOTOS

### DISCUSSION THREAD

**Discussion by Real-American-DAD** [d4rken-org/sdmaid-se#1128] [Q&A] [answered]
I like your app & even PURCHASED it (which I rarely do). 
> I'd love to see you add the capability of searching for
* "exact" VISUAL matches AND
* "similar" VISUAL matches, 
 for photos on my device. 
***
By simply clicking 1 button to search for both
& THEN having the option TO look at & choose which 
SIMILAR matches 
or
EXACT matches, 
I want to DELETE 
or keep. 
...
Thank you for your consideration of this NEW FEATURE.
... 
Looking forward to hear what you think.

**Comment by d4rken** [comment, marked as answer]
Don't these already exist? The deduplicator can already find exact matches by checksum and similar matches by perception hash. You might have to enable finding similar matches in the settings. I think it's off by default.

**Comment by Real-American-DAD** [comment]
Sorry, I was unaware of this setting.
Thank you ! 👍🏻
